src/main/webapp/intro.html

```diff
@@ -0,0 +1,284 @@
+                        <a class="social-icon text-xs-center" target="_blank" href="https://www.fiverr.com/share/qb8D02">
+                          <i class="fa fa-facebook"></i>
+                        </a>
+                      </li>
+                      <li class="list-inline-item">
+                        <a class="social-icon text-xs-center" target="_blank" href="https://www.fiverr.com/share/qb8D02">
+                          <i class="fa fa-twitter"></i>
+                        </a>
+                      </li>
+                    </ul>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <!-- ./Xin -->
+        <!-- Uri -->
+        <div class="col-xs-12 col-sm-6 col-md-4">
+          <div class="image-flip" ontouchstart="this.classList.toggle('hover');">
+            <div class="mainflip">
+              <div class="frontside">
+                <div class="card">
+                  <div class="card-body text-center">
+                    <p><img class=" img-fluid" src="images/Uri.png"  alt="card image"></p>
+                    <h4 class="card-title">Uri Herszbaum-harding</h4>
+                    <p class="card-text">Descibe Uri in a sentence or less. IAmAPlaceholder IAmAPlaceholder.</p>
+                  </div>
+                </div>
+              </div>
+              <div class="backside">
+                <div class="card">
+                  <div class="card-body text-center mt-4">
+                    <h4 class="card-title">Uri Herszbaum-harding</h4>
+                    <p class="card-text">Uri is a Junior at the University of Nebraska-Lincoln studying Computer Science. His hobbies include reading, writing, gaming and game development, as well as dodgeball.
+                    </p>
+                    <ul class="list-inline">
+                    <li class="list-inline-item">
+                      <a class="social-icon text-xs-center" target="_blank" href="https://www.fiverr.com/share/qb8D02">
+                        <i class="fa fa-google"></i>
+                      </a>
+                    </li>
+                      <li class="list-inline-item">
+                        <a class="social-icon text-xs-center" target="_blank" href="https://www.fiverr.com/share/qb8D02">
+                          <i class="fa fa-facebook"></i>
+                        </a>
+                      </li>
+                      <li class="list-inline-item">
+                        <a class="social-icon text-xs-center" target="_blank" href="https://www.fiverr.com/share/qb8D02">
+                          <i class="fa fa-twitter"></i>
+                        </a>
+                      </li>
+                    </ul>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <!-- ./Uri -->
+      </div>
+      <h5 class="section-title h1">Our Great Host</h5>
```

Comment:
NIT: Our Great Hosts

Alternatively
`Our Incredibly Talented, Hilarious, Intelligent Hosts` (just kidding, please don't put this :) )

Reply:
hahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahahhahahahahahahahah

Reply:
OMG i will not resolve this comments forever hhhh

Reply:
LOL. Too many adjectives, simply: Our perfect hosts. 💯